lang/src/lib.rs

```diff
@@ -215,6 +215,28 @@ impl Key for Pubkey {
     }
 }
 
+/// Defines the number of lamports in an account.
+pub trait Lamports {
```

Comment:
I think we should require clients to use `as_ref()` explicitly instead of adding more traits.